modules/tab_parameter_estimation/mixture_models/mixmod_em/mixmod_em.R

```diff
@@ -0,0 +1,169 @@
+mixmod_em_ui <- function(id) {
+  ns <- shiny::NS(id)
+
+  model_name <- "mix_mod_em"
+
+  shiny::fluidRow(
+    shiny::column(
+      width = 6,
+      bs4Dash::box(
+        width = NULL,
+        solidHeader = TRUE,
+        status = "primary",
+        title = "EM Algorithm",
+        mixmod_em_fun_ui(
+          id = ns("mixmod_em_fun"),
+          model_name = model_name
+        ),
+        htmltools::br(),
+        plot_prob_mix_fun_ui(
+          id = ns("plot_prob_mix_fun"),
+          model_name = model_name
+        ),
```

Comment:
We don't need `model_name` as input to mixmod_em and plot_prob_mix_fun as we can determine them using shinymetaVarname